core/templates/dev/head/pages/exploration_editor/simple_editor_tab/QuestionIdServiceSpec.js

```diff
@@ -18,16 +18,16 @@
 
  describe('QuestionId Service ', function() {
   var QuestionIdService;
-  var stubs ;
+  var stubs;
 
   beforeEach(function() {
     stubs = {
-      questionId : "Introduction",
-      label : "Hints",
-      subfieldId : "Introduction:Hints"
+      questionId: 'Introduction',
+      label: 'Hints',
+      subfieldId: 'Introduction:Hints'
     };
 
-    module('oppia'));
+    module('oppia');
   });
 
   beforeEach(inject(function($injector) {
```

Comment:
All tests are failing. Have you executed them before you committed and created PR?